pubsublite/admin.go

```diff
@@ -29,6 +29,18 @@ var (
 	errNoSubscriptionFieldsUpdated = errors.New("pubsublite: no fields updated for subscription")
 )
 
+// BacklogLocation refers to a location with respect to the message backlog.
+type BacklogLocation int
+
+const (
+	// End refers to the location past all currently published messages. End
+	// skips the entire message backlog.
+	End BacklogLocation = iota + 1
```

Comment:
Should there be an undefined BacklogLocation. If there was one you could default to a specified value if not set a by a user.

Reply:
The zero value was reserved for a future undefined/unspecified value, if required. For the create subscription option, we want users to specify either Beginning or End.